src/bin/cosmogony.rs

```diff
@@ -40,6 +40,11 @@ Accepted extensions are '.json', '.json.gz', '.jsonl', '.jsonl.gz'
         long = "disable-voronoi"
     )]
     disable_voronoi: bool,
+    #[structopt(
+        help = "Only generates labels for french and english",
```

Comment:
it's strange to hardcode the lang

shouldn't it be a `--lang=fr --lang=en` (or `--filter-lang`) ?

Reply:
I changed it so you can pick more languages. But if you don't use this option, no filter is performed.

Reply:
:+1: 

wouldn't it be more straightforward to use Option<Vec<String>> ?

Reply:
I might make it clearer that there are no filtering for empty param (but maybe just adding it in the comment is enough)